vwifi.c

```diff
@@ -295,47 +238,102 @@ static struct net_device_stats *owl_ndo_get_stats(struct net_device *dev)
     return &vif->stats;
 }
 
-/* Receive a packet: retrieve, encapsulate and pass over to upper levels */
+static netdev_tx_t owl_ndo_start_xmit(struct sk_buff *skb,
+                                      struct net_device *dev);
+
+/* Receive a packet: retrieve, encapsulate to skb, then do the following
+ * stuff based on the interface mode:
+ * STA mode: pass the skb to upper level (protocol stack).
+ * AP mode: do the following stuff based on the packet type:
+ *          1. unicast: if the skb is for another STA, then pass it
```

Comment:
Reduce the indention. 4 spaces should be enough.

Reply:
Sure.